assets/js/modules/analytics/common/account-field.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * AccountField component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CreateAccountField from './create-account-field';
+
+const AccountField = ( { hasError, accountName, setAccountName } ) => {
+	return (
+		<CreateAccountField
+			hasError={ hasError }
+			value={ accountName }
+			setValue={ setAccountName }
```

Comment:
Not sure there's much benefit in having field-specific props for each here.
* `accountName` could be `value`, and `setAccountName` could be `setValue`.
* `setValue` could be `onChange` and the API between components is more direct and clear.

Then all props could be passed through and you have less differences between each, and `CreateAccountField` is just a thin template component which is primarily concerned with DRYing up the markup for each.

Reply:
Agreed, I thought I had mentioned that before. Let's use `value` and `onChange` for all of these.

Reply:
Ah yes you did and I did part of this @felixarntz, will complete.

Reply:
Completed in https://github.com/google/site-kit-wp/pull/1389/commits/e97166b547e89d37cf74fe5c3d5f2b61ca6933fa